Source/NSubstitute.2010.sln.DotSettings

```diff
@@ -0,0 +1,4 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
Agreed I didn't spot that I had staged these files I will remove them before pushing again.  That being said it might be useful to have a .DotSettings file in the repo so that anyone else contributing that uses ReSharper a very common tool, use the same coding conventions.  The reason I created the file in the first place was because my own coding conventions are quite different and doing some of the things I do habitually made changes that did not fit with the existing conventions.  It does no harm if ReSharper is not installed on a developers computer and the file is there.